compute/resource_instance_pool.go

```diff
@@ -74,7 +74,7 @@ func ResourceInstancePool() *schema.Resource {
 			},
 			"max_capacity": {
 				Type:     schema.TypeInt,
-				Required: true,
+				Optional: true,
```

Comment:
It should say Optional in the docs as well (that was the merge conflict), it's saying Required now